README.md

```diff
@@ -139,7 +140,7 @@ So the similar result (but not rounded) can be gained with the usage of it. Chec
 ```haskell
 main :: IO ()
 main = do
-    putStrLn $ "The result:   " ++ formatHours 140
+    putStrLn $ "The result:   " ++ formatHours (Time 140)
```

Comment:
I think it's slightly better for documentation to use `hour` function instead of `Time` constructor